src/main/features/dashboard/views/macro/claimStatus/offerSettlement.njk

```diff
@@ -0,0 +1,24 @@
+{% macro offerSettlementReachedClaimDetails(claim) %}
+  <h2 class="heading-medium">{{ t('Agreement signed') }}</h2>
+  <p>
+    {{ t('You’ve both signed a legal agreement. The claim is now settled.') }}
+  </p>
+
+  <p>
+    {{ downloadTheAgreement(claim, DashboardUrlHelper.getStatusUrl(claim)) }}
+  </p>
+{% endmacro %}
+
+{% macro offerSettlementReachedDashboard(claim) %}
+
```

Comment:
either or - please add empty lines at the beginning and an end of all macros in this file or in none.

REQ.